src/battle_game/core/unit.h

```diff
@@ -6,9 +6,22 @@ namespace battle_game {
 
 class Bullet;
 
+class LifeBar {
+ public:
+  LifeBar();
+  bool display_{true};
+  glm::vec2 offset_{};
+  float length_{2.4f};
+  glm::vec4 front_color_{};
+  glm::vec4 background_color_{};
+  float fadeout_health_;
+  glm::vec4 fadeout_color_{};
```

Comment:
I mean, you can just put them into unit class. Or there's something else you were worried about.